app/models/support_interface/cancel_application_form.rb

```diff
@@ -0,0 +1,17 @@
+module SupportInterface
+  class CancelApplicationForm
+    include ActiveModel::Model
+
+    attr_accessor :application_form
+
+    def save!
+      application_form.application_choices.includes(:course_option).where(status: %w[awaiting_references application_complete]).find_each do |application_choice|
```

Comment:
Are we using the course option for anything?

Reply:
I don't know! Bullet is complaining about this. I'll investigate.